content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/configure-limit-parsys/touchui-limit-parsys.js

```diff
@@ -124,7 +124,10 @@
         }
         var limit = parseInt(currentLimit);
         var children = getChildEditables(parsysEditable);
-        isWithin = children.length <= limit;
+        //in case is a paste action calculates the final number or paragraphs by adding how many are on the clipboard
```

Comment:
rather "Take into account also the number of components in the clipboard in case this is a "paste"